src/containers/Token/TokenTransactionsTable/index.js

```diff
@@ -69,7 +70,7 @@ export const TokenTxTable = props => {
               </div>
             </div>
             <div className={`col-type tx-type ${tx.type}`}>
-              <TxLabel type={tx.type} t={t} />
+              <TxLabel type={tx.type} />
```

Comment:
how come we don't need to pass `t` as a prop anymore? I'm not seeing it used elsewhere here

Reply:
I updated TxLabel to use the `useTranslation` hook so it wont need to be passed.  I will have more PRs in the future updating more places.  That hook was added in the major version of react-i18next we are updating to.